package.json

```diff
@@ -62,5 +62,164 @@
     "hooks": {
       "pre-commit": "npm test"
     }
-  }
+  },
+  "files": [
+    "src/helpers/attachto.ts",
```

Comment:
personal opinion, but this feels a bit ugly. When someone adds a new file, now we have many different places to keep track of. Doesn't quite fit well to the principle of DRY (don't repeat yourself)

Reply:
out of curiosity, do other packages do this too ?

Reply:
In this blog post from September 2017 npm recommends this. They write:

> Only 57,000 packages use this method of controlling what goes into them, probably because it requires you to take inventory of your package. It’s by far the safest way to do it, though.

I don't mind maintaining this list whitelist of build assets. I feel there are greater challenges than that.